src/main/scala/org/broadinstitute/dsde/workbench/sam/openam/AccessPolicyDAO.scala

```diff
@@ -23,12 +23,12 @@ trait AccessPolicyDAO {
   def overwritePolicyMembers(id: FullyQualifiedPolicyId, memberList: Set[WorkbenchSubject]): IO[Unit]
   def overwritePolicy(newPolicy: AccessPolicy): IO[AccessPolicy]
   def listPublicAccessPolicies(resourceTypeName: ResourceTypeName): IO[Stream[ResourceIdAndPolicyName]]
-  def listPublicAccessPolicies(resource: FullyQualifiedResourceId): IO[Stream[AccessPolicy]]
+  def listPublicAccessPoliciesWithoutMembers(resource: FullyQualifiedResourceId): IO[Stream[AccessPolicyWithoutMembers]]
```

Comment:
nit: no strong opinion, but this name is no longer symmetric with listAccessPoliciesForUser. It's a little funny that one but not both of them has WithoutMembers.

Reply:
I think remove without members from function name, return type is explicit